python/lsst/analysis/tools/actions/plot/propertyMapPlot.py

```diff
@@ -0,0 +1,443 @@
+                    )
+                    # Add a distinct outline around the text for better
+                    # visibility in various backgrounds.
+                    zoomText.set_path_effects(
+                        [
+                            mpl_path_effects.Stroke(linewidth=4, foreground="white", alpha=0.8),
+                            mpl_path_effects.Normal(),
+                        ]
+                    )
+                    # Get the indices of pixels in the zoomed-in region.
+                    pos = mapData.valid_pixels_pos()
+                    # Reversed axes consideration.
+                    xmin, xmax = sorted([x0, x1])
+                    idx = (pos[0] > xmin) & (pos[0] < xmax) & (pos[1] > y0) & (pos[1] < y1)
+                    zoomIdx.append(idx[goodValues])
+
+            # Calculate weights for each bin to ensure that the peak of the
+            # histogram reaches 1.
+            weights = np.ones_like(values) / np.histogram(values, bins=nBinsHist)[0].max()
+
+            # Compute full-tract histogram and get its bins.
+            bins = ax2.hist(
+                values,
+                bins=nBinsHist,
+                label="Full Tract",
+                color=zoomColors[0],
+                weights=weights,
+                alpha=0.7,
+            )[1]
+
+            # Align the histogram (top right panel) with the skyproj plots.
+            pos1 = spf._ax.get_position()  # Top left.
+            pos4 = sp._ax.get_position()  # Bottom right.
+            cbarWidth = cbar.ax.get_position().height / colorBarAspect
+            # NOTE: cbarWidth != cbar.ax.get_position().width
+            ax2.set_position([pos4.x0, pos1.y0, pos4.width + cbarWidth, pos1.height])
+
+            # `exposure_time` histograms are bar-like, not suited for
+            # overplotting, so we exclude them.
+            if "exposure_time" not in mapName:
+                # Overplot the histograms for the zoomed-in plots.
+                for zoomFactor, zidx, color in zip(zoomFactors, zoomIdx, zoomColors[1:]):
+                    weights = np.ones_like(values[zidx]) / np.histogram(values[zidx], bins=bins)[0].max()
+                    histLabel = f"{self.prettyPrintFloat(zoomFactor)}X Zoom"
+                    ax2.hist(
+                        values[zidx],
+                        bins=bins,
+                        label=histLabel,
+                        color=color,
+                        weights=weights,
+                        alpha=0.6,
+                    )
+
+            # Set labels and legend.
+            ax2.set_xlabel(label)
+            ax2.set_ylabel("Normalized Count")
+            legend = ax2.legend(loc="best", frameon=False, fontsize=15)
+            for line, text in zip(legend.legendHandles, legend.get_texts()):
+                text.set_color(line.get_facecolor())
+
+            # Add nside info to `plotInfo`.
+            plotInfo["nside_coverage"] = mapData.nside_coverage
```

Comment:
The nside_coverage is an implementation detail of the map, and shouldn't be plotted.